### PR TITLE
Update outdated documentation in examples

### PR DESCRIPTION
**Pull request by davidlmobley**
@bannanc , can you do a quick check? Just fixes some outdated documentation.

### CODE REVIEW

**Comment by bannanc** [comment]
@davidlmobley 
I approved the edits, but now the tests are failing, it looks like it can't find the license is there anyway it has expired?
Here's the end of the check:
```
The command "conda install --yes nose nose-timer" exited with 0.
2.02s$ cd devtools && nosetests -vv --nocapture --with-timer $PACKAGENAME && cd ..
nose.config: INFO: Ignoring files matching ['^\\.', '^_', '^setup\\.py$']
Test ChemicalEnvironment objects with complicated torsion ... LICENSE: Could not open license file specified by OE_LICENSE environment variable "/home/travis/oe_license.txt"
LICENSE: Could not open license file "oe_license.txt" in local directory
LICENSE: N.B. OE_DIR environment variable is not set
LICENSE: No product keys!
LICENSE: No valid license found for oechem
```

**Comment by davidlmobley** [comment]
License SHOULDN'T be outdated, it was updated three months ago. Investigating.

**Comment by davidlmobley** [comment]
Yeah, looks like @jchodera  updated three months ago. Though, I don't know when his license expired. I closed and re-opened the PR to re-trigger the test; if it still fails, @bannanc , let me know and I can replace his license with ours (which has an expiration date I know) and see if it helps.

@jchodera what's the expiration date on your license? (Maybe this info should go in the README.md or similar.)

**Comment by bannanc** [comment]
@davidlmobley it failed again. 

I made a commit to your branch, that wouldn't trigger a problem would it?

**Comment by davidlmobley** [comment]
Nope, that wouldn't do it.

**Comment by davidlmobley** [comment]
I'm working on a different branch right now so I'll have to revisit this later or tomorrow.

**Comment by davidlmobley** [comment]
@bannanc - pushed an updated license; we'll see if that does the trick. 

For the record here [the procedure](https://docs.travis-ci.com/user/encrypting-files/) is actually super simple:
- set up travis locally, something like `sudo gem install travis`
- login to travis, e.g. `travis login --auto`
- copy normal OpenEye license to directory, then `travis encrypt-file oe_license.txt`
- Get resulting info and use it to update the `.travis.yml`
- Commit the encrypted license and updates to yml

**Comment by davidlmobley** [comment]
OK, so that's not the issue.

**Comment by davidlmobley** [comment]
@bannanc - I poked around a little but don't see an obvious issue, especially since it seems to have broken because of something someone else changed.

One thought is that perhaps OpenEye rolled out a new beta release which is broken on this platform; particularly, before the license error, there is a previous error relating to `import openeye`. You might try switching the `.travis.yml` to pip install the main OpenEye rather than the beta and see if that fixes things.

**Comment by davidlmobley** [comment]
(I would do it but have something else to deal with for a while.)

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ install:
   - source devtools/travis-ci/install.sh
   - export PYTHONUNBUFFERED=true
   # Unpack encrypted OpenEye license file
-  - if [ "$TRAVIS_SECURE_ENV_VARS" == true ]; then openssl aes-256-cbc -K $encrypted_5f58236b8908_key -iv $encrypted_5f58236b8908_iv -in oe_license.txt.enc -out $OE_LICENSE -d; fi
+  - if [ "$TRAVIS_SECURE_ENV_VARS" == true ]; then openssl aes-256-cbc -K $encrypted_4fb296b25e14_key -iv $encrypted_4fb296b25e14_iv -in oe_license.txt.enc -out oe_license.txt -d; fi
```
Comment (bannanc) [inline .travis.yml]: The deleted line line had `-out $OE_LICENSE` and the new line has `-out oe_license.txt` I'll try going back to the `$OE_LICENSE`

Comment (davidlmobley) [reply .travis.yml]: Ah, yes, possibly I broke it while fixing it.

Comment (bannanc) [reply .travis.yml]: well that didn't do it...

**Comment by bannanc** [comment]
So... #77 is passing, what is wrong with this one?

**Comment by davidlmobley** [comment]
Well, that's very odd. Perhaps it's an issue at the OpenEye end where they had a broken version for this platform or something. Let me close and re-open (triggering a re-run of the tests) and see what happens.

**Comment by davidlmobley** [comment]
Well, no. That didn't do it. Veeery strange.

Let me actually just pull the changed text stuff into a separate branch, open a new PR, and see what happens there.

**Comment by davidlmobley** [comment]
Closing, superseded by #79 .

**Comment by bannanc** [comment]
Well that was strange, glad new PR worked I guess.

**Pull request closed without merging**